includes/personalize.classes.inc

```diff
@@ -133,6 +133,18 @@ interface PersonalizeAgentInterface {
   public function stopNow();
 
   /**
+   * Whether this agent should be started now.
+   *
+   * An agent class can imlement some logic to determine whether or not the agent
+   * in question should be set to the "running" status.  For example, an agent
+   * type might want to start automatically when certain setup has been created.
+   *
+   * @return boolean
+   *   TRUE if the agent should be set to running, FALSE otherwise.
+   */
+  public function startNow();
```

Comment:
Hmmm, this has me thinking - maybe we could do all the verification here for Lift agents so that they start automatically once they've passed verification. Totally not related to this issue, just thinking out loud, so to speak :)